src/site/content/en/blog/fetch-upload-streaming/index.md

```diff
@@ -0,0 +1,359 @@
+---
+layout: post
+title: Streaming requests with the fetch API
+authors:
+  - jakearchibald
+description: >
+  Chrome 85 has an experimental implementation of streaming uploads, which you
+  can use on live sites using the origin trial.
+origin_trial:
+  url: https://developers.chrome.com/origintrials/#/view_trial/-7680889164480380927
+date: 2020-07-22
+updated: 2020-07-22
+hero: hero.jpg
+alt: A conoe pointed up stream.
+---
+
+Chrome 85 has an experimental implementation of request streams, meaning you can
+start making a request before you have the whole body available.
+
+You could use this to:
+
+- Warm up the server. In other words, you could start the request once the user
+  focuses a text input field, and get all of the headers out of the way, then
+  wait until the user presses 'send' before sending the data they entered.
+- Gradually send data generated on the client, such as audio, video, or input data.
+- Recreate web sockets over HTTP.
+
+But, like all low-level web platform features, I'm more excited about the
+ideas I haven't thought of myself!
```

Comment:
```suggestion
But since this is a low-level web platform feature, don't be limited by _my_ ideas. 
Maybe you can think of a much more exciting use-case for request streaming.
```

Reply:
I accidentally overwrote your improvement. Here I'm putting it back.